webrtc.go

```diff
@@ -105,17 +108,51 @@ func newSession(config SessionConfig) (*Session, error) {
 		scopedLogger.Warn().Err(err).Msg("Failed to create PeerConnection")
 		return nil, err
 	}
+
 	session := &Session{peerConnection: peerConnection}
 	session.rpcQueue = make(chan webrtc.DataChannelMessage, 256)
+	session.hidQueue = make(chan webrtc.DataChannelMessage, 1024)
```

Comment:
That's a lot of backlog... might be a bit excessive?

Reply:
Yes, given that we're introducing a very short timeout for hid write, setting it to `1024` here seems unnecessary.